plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2965,6 +2971,10 @@ blockly.warnings.singular_math=One of math blocks is empty. Using value 0 for it
 blockly.warnings.empty_compare_text=One of text compare blocks is empty. Using default value for it.
 blockly.warnings.empty_text_block=Empty text block. Its value will be null.
 blockly.warnings.empty_text_length=Empty text length block. Its value will be 0.
+blockly.warnings.empty_text_is_empty_block=Empty text is empty block. Its value will be true.
```

Comment:
```suggestion
blockly.warnings.empty_text_is_empty_block=Empty is text empty block. Its value will always be true.
```

Reply:
Errors above this line use this template `Empty text 'block name' block`, so in this case what I did follows the template. The name of the block is `is empty`.

Reply:
I think the string is fine as is now